apps/src/StudioApp.js

```diff
@@ -499,17 +499,24 @@ StudioApp.prototype.init = function (config) {
 
 StudioApp.prototype.initProjectTemplateWorkspaceIconCallout = function () {
   if (getStore().getState().pageConstants.showProjectTemplateWorkspaceIcon) {
-    addCallouts([{
-      id: 'projectTemplateWorkspaceIconCallout',
-      element_id: '#projectTemplateWorkspaceIcon',
-      localized_text: msg.workspaceProjectTemplateLevel(),
-      qtip_config: {
-        position: {
-          my: 'top center',
-          at: 'bottom center',
-        },
-      },
-    }]);
+    // The callouts can't appear until the DOM is 100% rendered by react. The
+    // safest method is to kick off a requestAnimationFrame from an async
+    // setTimeout()
+    setTimeout(() => {
+      requestAnimationFrame(() => {
```

Comment:
I don't understand why this helps - any more than nesting two `setTimeout`s or just the longer `setTimeout` did.  Since this is sort of a leaf node in the rendering chain I don't think it's a particularly harmful workaround, but I wish I the root cause of why the target isn't settled at `componentDidMount` time.

Reply:
I also wouldn't expect to need both of these async methods. Surprised that just one wouldn't be sufficient.

Reply:
I agree.